sources/advertisements/VRM New Core/Controllers/VRMProcessingController.swift

```diff
@@ -33,12 +38,23 @@ final class VRMProcessingController {
                 dispatchedResults.contains(result) == false
             }.forEach { item, result in
                 
-                if currentGroup == nil || currentGroup?.items.contains(item) == false || timeout == .hard {
+                if currentGroup == nil ||
+                    currentGroup?.items.contains(item) == false ||
+                    timeout == .hard {
                     dispatch(VRMCore.timeoutError(item: item))
                 } else if case .inline(let vast) = result.vastModel {
                     dispatch(VRMCore.selectInlineVAST(item: item, inlineVAST: vast))
                 } else if case .wrapper(let wrapper) = result.vastModel {
-                    dispatch(VRMCore.unwrapItem(item: item, url: wrapper.tagURL))
+                    guard let count = scheduledVRMItems[item]?.count else {
+                        assertionFailure("try to unwrap item, which wasn't started")
```

Comment:
`fatalError` vs `assertionFailure`? What is the difference?

Reply:
the assertions are ignored in release builds